Header header
float64 batt1
float64 batt2
duration expected_time
